File: src/main/java/com/paragon/api/util/player/EntityFakePlayer.java
package com.paragon.api.util.player;

import com.paragon.api.util.Wrapper;
import net.minecraft.client.entity.EntityOtherPlayerMP;
import net.minecraft.world.World;

import java.util.UUID;

public final class EntityFakePlayer extends EntityOtherPlayerMP implements Wrapper {

    public EntityFakePlayer(final World worldIn) {
        super(worldIn, mc.player.getGameProfile());
        this.copyLocationAndAnglesFrom(mc.player);
        this.inventory.copyInventory(mc.player.inventory);
        mc.world.addEntityToWorld(-Integer.MAX_VALUE, this);
        this.entityUniqueID = UUID.randomUUID();
    }

    public void despawn() {
        mc.world.removeEntity(mc.world.getEntityByID(-Integer.MAX_VALUE));
    }

}

File: src/main/kotlin/com/paragon/api/util/player/EntityFakePlayer.kt
package com.paragon.api.util.player

import com.paragon.api.util.Wrapper
import net.minecraft.client.Minecraft
import net.minecraft.client.entity.EntityOtherPlayerMP
import net.minecraft.world.World

class EntityFakePlayer(worldIn: World?) : EntityOtherPlayerMP(worldIn, Minecraft.getMinecraft().player.gameProfile), Wrapper {

    init {
        copyLocationAndAnglesFrom(minecraft.player)
        inventory.copyInventory(minecraft.player.inventory)
        minecraft.world.addEntityToWorld(-Int.MAX_VALUE, this)
    }

    fun despawn() {
        minecraft.world.removeEntity(minecraft.world.getEntityByID(-Int.MAX_VALUE))
    }
}